### PR TITLE
frontend: Fix keepAlive accumulation.

### LINKED ISSUE #17: frontend: keep-alive interval accumulation
When channel token expires and a new token is generated the keep-alive interval is duplicated. This causes accumulation of keep-alive requests to the backend that can grow to several identical requests per second for a single client.

### PR DESCRIPTION
**Pull request by stefan-dimitrov**
Fixes #17

### CODE REVIEW

**Comment by mgenov** [comment]
dist folder should be added to .gitignore I suppose.

File: frontend/src/pushapi.spec.js [hunk 1]
```diff
@@ -366,6 +366,21 @@ describe('PushApi', function () {
       expect(connectMethod).toHaveBeenCalledWith(subscriber);
     });
 
+    it('not double keep-alive interval after channel expire', function () {
```
Comment (mgenov) [inline frontend/src/pushapi.spec.js]: Maybe something like: it('uses single keep-alive for all channels'...   sounds better.

**Comment by mgenov** [comment]
LGTM !

File: frontend/src/pushapi.spec.js (hunk 1 above)
Comment (stefan-dimitrov) [inline frontend/src/pushapi.spec.js]: Yes, sound better. Will change it.

**Comment by stefan-dimitrov** [comment]
Regarding the dist directory: bower works by referencing the github repo directly - it doesn't store prebuilt files, so the distribution files must be committed in the repo for bower to install them.

**Pull request merged**